packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/invoke-model.ts

```diff
@@ -285,5 +320,27 @@ export class BedrockInvokeModel extends sfn.TaskStateBase {
       }),
     };
   };
+
+  private getInputSource(props?: BedrockInvokeModelInputProps, inputPath?: string, isFeatureFlagEnabled?: boolean): string | undefined {
+    if (props?.s3Location) {
+      return `s3://${props.s3Location.bucketName}/${props.s3Location.objectKey}`;
+    } else if (isFeatureFlagEnabled && props?.s3InputUri) {
+      return props.s3InputUri;
```

Comment:
can we make it simple if `isFeatureFlagEnabled` return `props.s3InputUri` so regardless it has undefined value, or any other value that can be translated as false like an empty string

Reply:
In that case if we return an empty string, it will populated in the input as { } for s3URI which could lead to failure with invalid URI provided for Input field.

Reply:
so we need to validate the customer input that the empty `s3InputUri` is not a valid input